### PR TITLE
MGMT-3515 Disable ntp validation for bootstrap in place

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

**Review by eranco74 — COMMENTED** [review]
/lgtm

File: internal/host/validator.go [hunk 1]
```diff
@@ -534,6 +534,10 @@ func (v *validator) printBelongsToMajorityGroup(c *validationContext, status Val
 
 func (v *validator) IsNTPSynced(c *validationContext) ValidationStatus {
 
+	if swag.StringValue(c.cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone {
```
Comment (YuviGold) [inline internal/host/validator.go:537]: Maybe this should be a function? `ShouldCheckNTP`

File: internal/cluster/validator.go [hunk 2]
```diff
@@ -489,6 +489,10 @@ func (v *clusterValidator) printNetworkPrefixValid(c *clusterPreprocessContext,
 }
 
 func (v *clusterValidator) isNtpServerConfigured(c *clusterPreprocessContext) validationStatus {
+	if swag.StringValue(c.cluster.HighAvailabilityMode) == models.ClusterHighAvailabilityModeNone {
```
Comment (YuviGold) [inline internal/cluster/validator.go:492]: Maybe this should be a function? `ShouldCheckNTP`

File: internal/host/validator.go (hunk 1 above)
Comment (tsorya) [reply internal/host/validator.go:537]: why to? it means for every place i will need this validation i will need to add function if should?

Comment (YuviGold) [reply internal/host/validator.go:537]: It doesn't mean that, but actually having a generic dynamic switch-case wrapping all the validations, is required for profiling. :)

**Comment by eranco74** [comment]
/hold
Seems that we might want to keep this validation

**Pull request closed without merging**